### PR TITLE
[Translations] Modal: i18n support

### PR DESCRIPTION
**Pull request by HalvorHaugan**
### Component Checklist 📝

- [x] JSDoc
- [x] Examples
- [x] Documentation
- [x] Storybook
- [x] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [x] Component tokens (`@navikt/core/css/tokens.json`)
- [x] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [x] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [x] New component? CSS import (`@navikt/core/css/index.css`)
- [x] Breaking change? Update migration guide. Consider codemod.
- [x] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

File: @navikt/core/react/src/util/i18n/locales/en.ts [hunk 1]
```diff
@@ -37,4 +37,7 @@ export default {
       labelSuffix: "delete",
     },
   },
+  Modal: {
```
Comment (KenAJoh) [inline @navikt/core/react/src/util/i18n/locales/en.ts:48]: Further on the discussion around "generic" translations [mentioned here](https://github.com/navikt/aksel/pull/3281#discussion_r1818682329). Should more generic words like: Expand, Collapse, Open, Close, Next, Previous ++ be under a generic key?

File: @navikt/core/react/src/util/i18n/locales/en.ts [hunk 2]
```diff
@@ -37,4 +37,7 @@ export default {
...
+    close: "Close",
```
Comment (KenAJoh) [inline @navikt/core/react/src/util/i18n/locales/en.ts:49]:
```suggestion
    close: "Close dialog",
```
Would this be more descriptive? Or do most people not know the concept of a "dialog" so "close" is better?

Comment (HalvorHaugan) [reply @navikt/core/react/src/util/i18n/locales/en.ts:49]: We had "Close modal" before, but David complained about it and said it was redundant (and annoying iirc 😅). The SR announces that you are in a dialog, so I think it's implied that "close" closes the dialog.

When it comes to seeing people, I think you are right in that many people probably doesn't understand "dialog". But I think most people are familiar with windows (not the OS but the UI concept) and the X button in the corner of many of them.

File: @navikt/core/react/src/util/i18n/locales/en.ts (hunk 1 above)
Comment (KenAJoh) [reply @navikt/core/react/src/util/i18n/locales/en.ts:48]: Ignore this

**Review by KenAJoh — DISMISSED** [review]

**Pull request merged**